pkg/landscaper/installations/subinstallations/subinstallations.go

```diff
@@ -151,7 +153,13 @@ func (o *Operation) createOrUpdateNewInstallation(ctx context.Context, inst *lsv
 
 	if subInst == nil {
 		subInst = &lsv1alpha1.Installation{}
-		subInst.GenerateName = fmt.Sprintf("%s-", subInstTmpl.Name)
+
+		generateName := subInstTmpl.Name
```

Comment:
I think we should inaddition also restrict the subinstallation name.

Reply:
We have restricted the length of generateName of a subinstallation. There seems to be no direct setting of the name.